UKHomeOffice/nmsw-ui
# NMSW-32 Redirect to landing page on 404s

# Ticket

NMSW-32

----

## AC

Given I have typed in a page url that does not exist
When I hit enter
Then I am taken to the landing page

----

## To test

- run app locally
- go to localhost: 3000
- try to go to /badpage
- you should be taken to the landing page

----

## Notes
